hand_offs.md

```diff
@@ -0,0 +1,40 @@
+# Client Handoffs
+
+All Start
+1. Two weeks before or more!, write an email including PMs on both sides to set the date 
+and clarify any expectations around handoff at project end.
+1. Be sure you have a complete [readme](https://github.com/abtion/guidelines/blob/master/standard_readme.md) 
+incluing any diagrams or introductions to libraries that are heavily used as
+well as any known issues.
+
+Heroku apps
+1. Make sure all related Heroku plugins are pointed to client emails (ex:
+librato alerts or rollbar alerts)
+1. Make sure all PRs are closed and push to heroku, then transfer the pipeline 
+in the GUI. Client will setup their own github with what they pull from Heroku.
+
+
+PHP apps
+1. Transfer ownership in GitHub itself of all repos
+
+All Finish
+1. Talk to client PM to have them create any environment key/values needed for 
+the app to run where we had been using our dev accounts.
+- [  ] AWS
+- [  ] Mapping services?
+- [  ] Mail Service
+- [  ] Pingdom/Alerts
+- [  ] CI/CD stuff
+- [  ] Database URL and credentials?
+- [  ] Analytics accounts
+- [  ] Scheduler services?
+- [  ] SMS/Push services?
+- [  ] Product management backlog (Asana or Tracker?)
+1. Guide PM through secure transfer of .env file (if we are passing on accounts 
```

Comment:
It would be great to know how to securely transfer secrets. I'm not sure what a mapbox is. Probably not related to https://www.mapbox.com/

Reply:
Yeah, 100% is https://www.mapbox.com/. It's an example pulled from Gehl where we signed up for the service for them to get a mapping API key.

Reply:
Ah. Now, I understand. I found it a bit confusing, so perhaps other devs will find it confusing too.
```suggestion
1. Guide PM through a secure transfer of .env file (if we are passing on accounts with keys that will be transitioned to clients, we'll need to transfer the private keys for all the services we signed up for on their behalf)
```

Reply:
I think we should specify the way of sharing the .env file. 
The most practical/secure I can think of is:
1. Have `.env` encrypted using [Gitcrypt](https://github.com/AGWA/git-crypt). Store encryption key in 1Password
2. When handing over to a client, [invite the client as a guest to 1password](https://support.1password.com/guests-teams/). Grant permissions to see Gitcypt's encryption key only.

Reply:
What do you think about using the built in rails secrets vs gitcrypt? https://www.engineyard.com/blog/rails-encrypted-credentials-on-rails-5.2